python3/ltk/actions/move_action.py

```diff
@@ -55,8 +56,14 @@ def _do_move(self):
         try:
             if self.rename and self.source_type == 'file' and self.path_to_source.rstrip(self.path_sep).rstrip(self.doc['name']) != self.path_to_source.rstrip(self.path_sep):
                 new_name = os.path.basename(self.path_to_destination)
+                response = self.api.document_update(self.doc['id'], title=new_name)
+                if response.status_code == 423 and 'next_document_id' in response.json():
```

Comment:
So I made a comment to explain this, but it is because the update through the api and the local update are done separately, if we get a 402 or a 423, we want to prevent the rest of the changes from happening, if we got a 410 or a 202, then we want the move to go ahead and happen, I also put it inside of a Try Catch just in case something happened with the API request.